README.md

```diff
@@ -39,162 +39,4 @@ https://tldp.org/LDP/abs/html/
 - python3 is used for small tasks like unpivoting source data.
```

Comment:
@kalenikaliaksandr proposed to use everywhere **python** not python3

Reply:
replaced